llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -2000,6 +2000,14 @@ bool RISCVTargetLowering::shouldSinkOperands(
   if (!I->getType()->isVectorTy() || !Subtarget.hasVInstructions())
     return false;
 
+  // Don't sink splat operands if the target prefers it. Some targets requires
```

Comment:
This change is not mentioned in the patch description

Reply:
Updated.

Reply:
I took the changes out of this patch. I will work on the next patch once this lands since there are a few fixups here and it will be easier to work off the squashed commit.